docs/README.md

```diff
@@ -120,6 +120,18 @@ Installation process
       helm install csi-baremetal csi/csi-baremetal-deployment --set driver.drivemgr.type=halmgr \
       --set global.registry=$REGISTRY --set global.registrySecret=$DOCKER_REGISTRY_SECRET
       ```
+* Note about Upgrade
```

Comment:
Please put in **Upgrade process** section. Should it be done before or after `helm upgrade`?

Reply:
> Should it be done before or after helm upgrade?

No matter before or after, it must be done if CRDs update expected